src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -969,6 +969,9 @@ ucs_status_t uct_dc_mlx5_ep_fc_ctrl(uct_ep_t *tl_ep, unsigned op,
             goto out_dci_put;
         } else if (ret == UCS_KH_PUT_KEY_PRESENT) {
             fc_entry = &kh_value(&iface->tx.fc_hash, it);
+
+            /* Do not resend FC request if timeout is not reached, or window is
+             * still not empty */
```

Comment:
BTW, why need to check dc_ep->fc.fc_wnd > 0 ? i think we already check it before calling this function?